_episodes_rmd/04-ggplot2.Rmd

```diff
@@ -396,7 +396,7 @@ change in the code to put the boxplot layer in front of the jitter layer?
 >   respondent is a member of an irrigation association (`memb_assoc`).
 >
 > > ## Solution
-> > ```{r boxplot-exercise-factor}
+> > ```{r boxplot-exercise-factor, fig. alt = "Previous plot with dots colored based on whether respondent was a member of an irrigation association."}
```

Comment:
```suggestion
> > ```{r boxplot-exercise-factor, fig.alt = "Previous plot with dots colored based on whether respondent was a member of an irrigation association."}
```